lib/ain-evm/src/evm.rs

```diff
@@ -182,36 +188,90 @@ impl EVMServices {
+                deploy_contract_tx(get_instrinics_registry().contract.init_bytecode, &base_fee)?;
+            all_transactions.push(Box::new(tx));
+            receipts_v3.push((receipt, Some(address)));
+
+            // Deploy DFIIntrinsics contract
             let DeployContractInfo {
                 address,
                 storage,
                 bytecode,
-            } = intrinsics_contract(executor.backend, dvm_block_number, current_block_number)?;
+            } = intrinsics_contract_v1(dvm_block_number, current_block_number)?;
 
             trace!("deploying {:x?} bytecode {:?}", address, bytecode);
             executor.deploy_contract(address, bytecode, storage)?;
             executor.commit();
 
+            // DFIIntrinsics contract deployment TX
+            let (tx, receipt) = deploy_contract_tx(
+                get_intrinsic_contract_v1().contract.init_bytecode,
+                &base_fee,
+            )?;
+            all_transactions.push(Box::new(tx));
+            receipts_v3.push((receipt, Some(address)));
+
             // Deploy transfer domain contract on the first block
             let DeployContractInfo {
                 address,
                 storage,
                 bytecode,
-            } = transfer_domain_contract();
+            } = transfer_domain_contract_v1();
+
+            trace!("deploying {:x?} bytecode {:?}", address, bytecode);
+            executor.deploy_contract(address, bytecode, storage)?;
+            executor.commit();
+            let (tx, receipt) = deploy_contract_tx(
+                get_transferdomain_contract_v1().contract.init_bytecode,
+                &base_fee,
+            )?;
 
+            all_transactions.push(Box::new(tx));
+            receipts_v3.push((receipt, Some(address)));
+
+            // Deploy transfer domain proxy
+            let DeployContractInfo {
+                address,
+                storage,
+                bytecode,
+            } = transfer_domain_proxy(get_transferdomain_contract_v1().fixed_address)?;
             trace!("deploying {:x?} bytecode {:?}", address, bytecode);
             executor.deploy_contract(address, bytecode, storage)?;
             executor.commit();
-            let (tx, receipt) = transfer_domain_deploy_contract_tx(&base_fee)?;
 
+            // transfer domain proxy deployment TX
+            let (tx, receipt) =
+                deploy_contract_tx(get_transferdomain_proxy().contract.init_bytecode, &base_fee)?;
             all_transactions.push(Box::new(tx));
             receipts_v3.push((receipt, Some(address)));
+
+            // deploy DST20 implementation contract
+            let DeployContractInfo {
+                address,
+                storage,
+                bytecode,
+            } = dst20_implementation();
+            trace!("deploying {:x?} bytecode {:?}", address, bytecode);
+            executor.deploy_contract(address, bytecode, storage)?;
+            executor.commit();
```

Comment:
no pushing to all_transactions and receipts?

Reply:
Fixed: 7eab71b00ea429d82d815a621400772cc3aac8b1